File: Smodr/src/main/java/com/kevintcoughlin/smodr/services/FeedService.java
package com.kevintcoughlin.smodr.services;

import com.kevintcoughlin.smodr.models.Feed;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface FeedService {
    @GET
    Call<Feed> feed(@Url String url);
}

File: Smodr/src/main/java/com/kevintcoughlin/smodr/services/FeedService.kt
package com.kevintcoughlin.smodr.services

import com.kevintcoughlin.smodr.models.Feed
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface FeedService {
    @GET
    fun feed(@Url url: String?): Call<Feed?>?
}